.github/CODEOWNERS

```diff
@@ -0,0 +1,8 @@
+# Code owners file.
+# This file controls who is tagged for review for any given pull request.
+
+# Learn about CODEOWNERS file format:
+#  https://help.github.com/en/articles/about-code-owners
+#
+
+* @ThomsonTan @seemk @TomRoSystems @jsuereth @maxgolov @lalitb @pyohannes
```

Comment:
Can we add something like `@open-telemetry/cpp-contrib-approvers`?

Reply:
Have added this group now.